README.md

```diff
@@ -40,33 +39,36 @@ First, clone `disclosure-backend` (or your fork of it) to your own local copy.
 
   (you will have to activate this environment (or virtualenv) every time you want to start working)
 
-3. Install project requirements with:
-   `pip install -r requirements.txt`
+3. Install mysql
 
-4. install mysql 
+  OSX:
+   ```
+   brew install mysql
+   brew install libssl
+   ```
   * When prompted for a password, remember it because you'll need it.
 
-### MySQL
+4. Install project requirements with:
+   ```
+   pip install -r requirements.txt
+   ```
 
-#### Install mysql
 
-```
-    Mac: brew install mysql
-```
+### Database setup
 
-#### Prepare the database
+1. Create the database
+  ```
+  mysql -p --user root
+  mysql> create database calaccess_raw;
+  mysql> \q
+  ```
 
-```
-mysql -p --user root
-mysql> create database calaccess_raw;
-mysql> \q
-python manage.py migrate
-```
+2. Modify `settings.py` (or create `settings_local.py`)
```

Comment:
Just a tiny nit: I think we should reword this to only encourage `settings_local.py` because settings.py is checked in.